crates/fluvio-spu/src/control_plane/dispatcher.rs

```diff
@@ -136,6 +139,7 @@ impl ScDispatcher<FileReplica> {
 
                 _ = status_timer.next() =>  {
                     self.send_lrs_status_back_to_sc(&mut sink).await?;
```

Comment:
Hmm.  this is sending too many individual requests.  This should be batched so only a a single status back to SC

Reply:
I filtered with a hashset at least also applied a little of DRY (Don't Repeat Yourself)